src/zen/kbs/ZenKeyboardShortcuts.mjs

```diff
@@ -743,6 +743,19 @@ class nsZenKeyboardShortcutsLoader {
       )
     );
 
+    // Pin/Unpin tab toggle
+    newShortcutList.push(
```

Comment:
Please add this under the migration process, like with the other shortcuts

Reply:
fixed!